docs/Fields.md

```diff
@@ -1106,6 +1106,132 @@ export const PostShow = (props) => (
+    <TextField source="name">
+    <TextField source="description">
+</TranslatableFields>
+```
+
+You may override the locale selected by default using the `defaultLocale` prop which react-admin sets to the current UI locale when not provided.
+
+```jsx
+<TranslatableFields locales={['en', 'fr']} defaultLocale="fr">
+    <TextField source="name">
+    <TextField source="description">
+</TranslatableFields>
+```
+
+By default, `<TranslatableFields>` will allow users to select the displayed locale using Material-ui tabs with the locale code as their labels.
+
+You may override the tabs labels using translation keys following this format: 'ra.locales.locale_code'. For example `ra.locales.en` or `ra.locales.fr`.
+
+You may override the language selector using the `selector` prop which accept a React element:
+
+```jsx
+const Selector = () => {
+    const {
+        locales,
+        selectLocale,
+        selectedLocale,
+    } = useTranslatableContext();
+
+    const handleChange = (event): void => {
+        selectLocale(event.target.value);
+    };
+
+    return (
+        <select
+            aria-label="Select the locale"
+            onChange={handleChange}
+            value={selectedLocale}
+        >
+            {locales.map(locale => (
+                <option
+                    key={locale}
+                    value={locale}
+                    // This allows to correctly links the containers for each locale to their labels
+                    id={`translatable-header-${locale}`}
+                >
+                    {locale}
+                </option>
+            ))}
+        </select>
+    );
+};
+
+<TranslatableFields
+    record={record}
+    resource="products"
+    basePath="/products"
+    locales={['en', 'fr']}
+    selector={<Selector />}
+>
+    <TextField source="name" />
+    <TextField source="description" />
+</TranslatableFields>
+```
+
+If you have multiple `TranslatableFields` on the same page, you should specify a `groupKey` so that react-admin can create unique identifiers for accessibility.
+
+```jsx
+<TranslatableFields locales={['en', 'fr']} groupKey="essential-fields">
+    <TextField source="name">
+    <TextField source="description">
+</TranslatableFields>
+```
+
+### Using Translatable Fields In List or Show views
+
+The `TranslatableFields` component is not meant to be used inside a `List` as you probably don't want to have tabs inside multiple lines. The simple solution to display a translatable value would be to specify its source like this: `name.en`. However, you may want to display its translation for the current admin locale.
+
+In this case, you'll have to get the current locale through the `useLocale` hook and set the translatable field `source` dynamically.
+
+```jsx
+const PostList = (props) => {
+    const locale = useLocale();
+
+    return (
+        <List {...props}>
+            <Datagrid>
+                <TextField source={`name.${locale}`}>
+                <ReferenceArrayField
```

Comment:
the ReferenceField here dilutes the example. Could you simply use a TextField in the tags list?

Reply:
I wanted to show the sort field

Reply:
Are you sure you want to remove it?